mk_verificator/tests/system/migration/test_migration.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import mk_verificator.utils as utils
+
+
+@pytest.fixture
+def create_vm(nova_client):
+    config = utils.get_configuration(__file__)
+
+    image_id = config['image_ref']
+    flavor_id = config['flavor_ref']
+
+    net_id = config['network_ref']
+
+    nics = [{"net-id": net_id, "v4-fixed-ip": ''}]
```

Comment:
`v4-fixed-ip: ''`
is it required format? I'm not expert in nova_client and just asking why that specified as empty string